episodes/144/README

```diff
@@ -0,0 +1 @@
+ssh capv@10.176.37.216
```

Comment:
Do you need a README and a README.md? That might confuse what github shows.

Reply:
fixed !